packages/mjml-section/src/index.js

```diff
@@ -189,7 +189,7 @@ class Section extends Component {
   renderSection () {
     const { renderWrappedOutlookChildren, mjAttribute, children, parentWidth } = this.props
     const fullWidth = this.isFullWidth()
-    const divProps = fullWidth ? {} : {
+    const divProps = {
```

Comment:
Hi, sorry this is needed to avoid duplicate css-class when the section is in full width